packages/ckeditor5-core/src/editor/editorui.js

```diff
@@ -172,6 +179,46 @@ export default class EditorUI {
 		return this._editableElementsMap;
 	}
 
+	/**
+	 * Returns viewport offsets object
```

Comment:
Use backticks to wrap code fragments.

Reply:
Missing `:` at the end. Also, the code snippet deserves to be a block snippet, so a triple backtick.

tldr something like:

````
Returns viewport offsets object:

```js
{ ... }
```

Only something.
````